berth_reservations/tests/conftest.py

```diff
@@ -10,3 +10,4 @@ def autouse_django_db(db):
 def force_settings(settings):
     settings.MAILER_EMAIL_BACKEND = "django.core.mail.backends.locmem.EmailBackend"
     settings.DEFAULT_FROM_EMAIL = "noreply@foo.bar"
+    settings.NOTIFICATIONS_ENABLED = 1
```

Comment:
this is Python syntax now, instead of the dotenv syntax, so `True` wouldbe more appropriate 🙂